CONTRIBUTING_DOCS.md

```diff
@@ -93,57 +91,77 @@ Let's say there's a single commit that makes changes to the `master` directory w
 
 ## Doc site organization
 
-### Overview
-
-The docs (currently) are split into four main sections.
+The docs are divided into the following sections:
 
 - [Introduction](#introduction)
-- [Getting started](#getting-started)
-- [Usage](#usage)
+- [Install](#install)
+- [Operations](#operations)
+- [Networking](#networking)
+- [Security](#security)
 - [Reference](#reference)
 
+With the except of Introduction and Reference, content should be tasked-based or "how to". All top-level titles in topics should use an action verb (e.g. configure, enable, modify) with initial caps. For example:
```

Comment:
"Except for", or "With the exception of"

Reply:
"task-based"

Reply:
fixed.